### PR TITLE
Changed wording on certificates and added proper navigation label hlighting

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit aff577ea53099244ae36fd23ca12372d584583a2: Broken link result: [!] Error(s) found by the Links validator: 5 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06e-Testing-Cryptography.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Given the continuous evolution of all third party libraries, this should not be
 - **Find the library being used**: This can be done using the following methods:
   - Check the [cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#cartfile "cartfile") if Carthage is used.
   - Check the [podfile](https://guides.cocoapods.org/syntax/podfile.html "podfile") if Cocoapods is used.
-  - Check the linked libraries: Open the xcodeproj file and check the project properties. Go to the tab "Build Phases" and check the entries in "Link Binary With Libraries" for any of the libraries. See earlier sections on how to obtain similar information using [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF "MobSF").
+  - Check the linked libraries: Open the xcodeproj file and check the project properties. Go to the tab **Build Phases** and check the entries in **Link Binary With Libraries** for any of the libraries. See earlier sections on how to obtain similar information using [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF "MobSF").
```
Comment (cpholguera) [inline Document/0x06e-Testing-Cryptography.md]:
```suggestion
  - Check the linked libraries: Open the xcodeproj file and check the project properties. Go to the **Build Phases** tab and check the entries in **Link Binary With Libraries** for any of the libraries. See earlier sections on how to obtain similar information using [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF "MobSF").
```

File: style_guide.md [hunk 2]
```diff
@@ -339,11 +339,11 @@ $ adb pull <remote_file> <target_destination>
 When they do not occur in a code block, place the following code-related keywords in backticks (``), double straight quote marks (""), or leave unpunctuated according to the table:
 
 |    Backticks    | Quotation Marks | No Punctuation |
-| --------------- | --------------- | -------------- |
+| --- | --- | --- |
 | function names  |  section titles | application name |
 |   method names  |  chapter titles | folder names  |
-|    commands     |    book titles  | navigation (e.g., Home -> Menu)|
-|   class names   | flags values (e.g., "true", lowercase) | ID #s |
+|    commands     |    book titles  | ID #s |
```
Comment (cpholguera) [inline style_guide.md]: ID #s was already there but I cannot tell what it is. We should make it clear

File: Document/0x05c-Reverse-Engineering-and-Tampering.md [hunk 3]
```diff
@@ -818,7 +818,7 @@ Moving down a level in the OS hierarchy, you arrive at privileged functions that
 
 Strace is a standard Linux utility that monitors interaction between processes and the kernel. The utility is not included with Android by default, but can easily be built from source via the Android NDK. Strace is a very convenient way to monitor a process' system calls. Strace depends, however on the `ptrace` system call to attach to the target process, so it only works up to the point at which anti-debugging measures start up.
 
-If the Android "stop application at startup" feature is unavailable, you can use a shell script to launch the process and immediately attach strace (not an elegant solution, but it works):
+If the Android **stop application at startup** feature is unavailable, you can use a shell script to launch the process and immediately attach strace (not an elegant solution, but it works):
```
[the remark this answers is not in the document]
Comment (commjoen) [reply Document/0x05c-Reverse-Engineering-and-Tampering.md]: leaving it for now.

**Comment by OMTGreleaser** [comment]
Results for commit 4ba746fefc9b6f13f8c96708a9dfb34409bf06f0: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: style_guide.md (hunk 2 above)
Comment (commjoen) [reply style_guide.md]: So the IDs came in 2 years ago... I totally forgot about it... maybe we should remove it?

**Comment by OMTGreleaser** [comment]
Results for commit 5f4d02e04902b6866c2cc144e4afd90049664073: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**